pubspec.yaml

```diff
@@ -12,11 +12,12 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_background: ^1.3.0+1
-  flutter_webrtc: ^0.12.1+hotfix.1
+  flutter_webrtc: ^0.12.3
   google_fonts: ^6.2.1
   http: ^1.2.2
   intl: ^0.19.0
-  livekit_client: ^2.3.0
+  livekit_client:
+    path: ../client-sdk-flutter
```

Comment:
this should be undone, right?

Reply:
yeah, when client-sdk is released to pub, I can update this